clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -93,24 +115,28 @@ object ElasticsearchIndex extends ModelAutoDerivation {
   val WgsUbam: ElasticsearchIndex[WgsUbamIndex.type] =
     new ElasticsearchIndex(
       WgsUbamIndex,
+      UbamMetadata().asJson,
       ElasticsearchFieldMapper.StringsToTextFieldsWithSubKeywords
     )
 
   val Gvcf: ElasticsearchIndex[GvcfIndex.type] =
     new ElasticsearchIndex(
       GvcfIndex,
+      GvcfMetadata(dataType = Option(DataType.WGS)).asJson,
       ElasticsearchFieldMapper.StringsToTextFieldsWithSubKeywords
```

Comment:
Needs to include `data_type` as an additional key field, if we're not deriving it from defaults.